File: URLShortener.Web/app/interfaces/url.info.ts
export interface IUrlInfo {
    Id: number;
    Url: string;
    ShortUrl: string;
    CreateAt: number;
    ClicksCount: number;
}

//export class IUrlInfo implements IUrlInfo {
//    constructor(public Id: number,
//        public Name: string,
//        public Key: string,
//        public CreateAt: number,
//        public ClicksCount: number){
//    }
//}

File: URLShortener.WebAPIService/Models/UrlInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace URLShortener.WebAPIService.Models
{
    public class UrlInfo
    {
        public long Id { get; set; }
        public string Url { get; set; }
        public string ShortUrl { get; set; }
        public DateTime CreateAt { get; set; }
        public long ClicksCount { get; set; }
    }
}
